src/components/Setup/Views/InstallPWA.tsx

```diff
@@ -116,13 +103,12 @@ const InstallPWA = () => {
     }
 
     const IOSInstructions = () => (
-        <div className="flex flex-col gap-4">
-            <div>
-                <StepTitle text="Step 1: Open the Sharing Options" />
-                <p>
-                    Tap the share icon (<ShareIcon />) at the bottom of your screen and then "Add to Home Screen"
-                </p>
-            </div>
+        <div className="space-y-4">
+            <StepTitle text="Add Peanut wallet in to your home screen" />
+            <p>
+                To add Peanut wallet to your Home screen, tap the (<ShareIcon />) icons and then {`"Add home Screen"`}{' '}
```

Comment:
Q: why did this copy change? is this how it is in figma? I don't think we should say "peanut wallet"

Reply:
yes I picked it up from figma